grobid-core/src/test/java/org/grobid/core/engines/FullTextParserTest.java

```diff
@@ -43,6 +55,41 @@ public static void tearDown() {
         GrobidFactory.reset();
     }
 
+    public DocumentPiece getWholeDocumentPiece(Document doc) {
```

Comment:
maybe `getWholeDocumentPiece` and `getWholeDocumentParts` could be moved to a central place, if it doesn't exist already.